geojson_modelica_translator/modelica/modelica_project.py

```diff
@@ -89,13 +91,24 @@ def _load_data(self) -> None:
         """method to load all of the files into a data structure for processing"""
         # walk the tree and add in all the files
         for file_path in self.root_directory.rglob('*'):
+            if file_path.suffix in self.file_types_to_skip and file_path.is_file():
+                # skip files that have the file_types_to_skip suffix
+                continue
+
             if file_path.suffix in self.file_types and file_path.is_file():
                 # only store the relative path that is in the package
                 rel_path = file_path.relative_to(self.root_directory)
                 self.file_data[str(rel_path)] = ModelicaFileObject(file_path)
             elif file_path.is_dir():
-                # this is a directory, just add in
-                # a temp object for now to keep the path known
+                # this is a directory, add in an empty ModelicaFileObject
+                # to keep track of the directory.
+                #
+                # however, we ignore if there is a tmp directory or the parent dir is
+                # tmp. Maybe we need to support more than 2 levels here.
+                if file_path.name == 'tmp' or file_path.parent.name == 'tmp':
```

Comment:
```suggestion
                if 'tmp' in file_path.parts:
```

This will pick up 'tmp' anywhere in the path. I think that is the desired behavior? Bless you for commenting your code!